apps/pie-docs/src/components/notification/overview/overview.md

```diff
@@ -0,0 +1,401 @@
+            width: 524
+        }]
+    }
+} %}
+
+### Title
+
+The title can be removed where it is not necessary.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/modifier-title.svg",
+    alt: "A notification with only a trailing icon, supporting text and dismiss button",
+    width: 1024
+} %}
+
+### Actions
+
+#### Dual actions
+
+When two separate actions are required.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/modifier-actions-dual-actions.svg",
+    alt: "A notification with a title, a trailing icon, supporting text, dismiss button and two action buttons",
+    width: 1024
+} %}
+
+#### Single action
+
+When a single action is required.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/modifier-actions-single-action.svg",
+    alt: "A notification with a title, a trailing icon, supporting text, dismiss button and one action button",
+    width: 1024
+} %}
+
+#### No actions
+
+When the user is not required to take action against the notification. 
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/modifier-actions-no-actions.svg",
+    alt: "Anatomy of a notification.",
+    alt: "A notification with a title, a trailing icon, supporting text, dismiss button and no action buttons",
+    width: 1024
+} %}
+
+---
+
+## Usage
+
+
+{% componentDetailsTable {
+  tableData: usage
+} %}
+
+---
+
+## Content
+
+Notifications provide limited space for content, and therefore the content must be short and concise. A user should be able to quickly scan the notification, be apprised of the situation, and know what to do next.
+
+### Title
+
+- The title should be short and descriptive, explaining the most important piece of information.
+- When possible, communicate the main message using just the title.
+
+### Supporting text
+
+- Be concise and avoid repeating or paraphrasing the title.
+- Limit content to one or two short sentences.
+
+---
+
+## Interactions
+
+### Close icon
+
+When the user clicks the icon button, the notification is dismissed.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/interaction-close-icon.svg",
+    alt: "A notification component with a dismiss button",
+    width: 1024
+} %}
+
+### Actions
+
+The actions in the notification can do things like redirecting the user. This might result in the notification closing once a task is done, or it could directly close the notification.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/interaction-actions.svg",
+    alt: "A notification component with a user action buttons",
+    width: 1024
+} %}
+
+---
+
+## Narrow
+
+All props are available in narrow screen sizes.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/notification/narrow.svg",
+    alt: "The notification component on narrow screens",
```

Comment:
On the designs, narrow and compact images are aligned left (although I do think they look better center-aligned, maybe just worth syncing up with Rachel or Molly for a sign off?)

Reply:
I brought this up with @RLumleyJET and we agreed to keep them centered. 💯